src/NuGet.Core/NuGet.LibraryModel/FrameworkDependencyFlagsUtils.cs

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace NuGet.LibraryModel
+{
+    public class FrameworkDependencyFlagsUtils
+    {
+        public static FrameworkDependencyFlags Default = FrameworkDependencyFlags.None;
```

Comment:
`readonly`.  Does it need to be `public` or will `internal` suffice?  The same question also applies to the class.